tasks/metrics/ci.rake

```diff
@@ -3,6 +3,7 @@ task ci: %w[ci:metrics metrics:mutant]
 
 namespace :ci do
   tasks = %w[
+    spec:integration
```

Comment:
Maybe it would help to place this second in the task prerequisites? This way CI fails much faster if unit tests fail